regions_www/m/pct_shiny_funs.R

```diff
@@ -3,15 +3,16 @@
 #################
 
 ## Region names [NB copy of this also in pct-scripts/00_setup_and_funs - if modify here, modify there too]
-get_pretty_region_name <- function(region_name, the = T){
-  if (the == T) {
-    region_name <- gsub("isle-of-wight", "the-isle-of-wight", region_name, perl=TRUE)
-    region_name <- gsub("north-east", "the-north-east", region_name, perl=TRUE)
-    region_name <- gsub("west-midlands", "the-west-midlands", region_name, perl=TRUE)
-  }
-  region_name <- gsub("(^|-)([[:alpha:]])", " \\U\\2", region_name, perl=TRUE)
+get_pretty_region_name <- function(region_name_in, the = T){
+  regions_thes <- c(
+    "isle-of-wight" = "The Isle of Wight",
```

Comment:
Sure, I've re-added the "The" ability, I was only doing the change for this repo and hadn't thought about modifying `pct-script`.


Before this change
```R
get_pretty_region_name("isle-of-wight")
=> " the Isle of Wight"
```
(lower case "the" and an unwanted space at the start)

With this change:
```R
get_pretty_region_name("isle-of-wight")
=> "The Isle of Wight"
```
so I don't really follow about part about capitalisation. Would you mind giving an example of how this isn't working?

It still works for regions like:
```R
get_pretty_region_name("cornwall-and-isles-of-scilly")
=> "Cornwall and Isles of Scilly"
``

Reply:
@nikolai-b , the issue with capitalisation is that I think  
get_pretty_region_name("isle-of-wight") => "the Isle of Wight" 
was correct and 
get_pretty_region_name("isle-of-wight") => "The Isle of Wight"
is wrong.  The 'the' on Isle of Wight (or the other 2 edge regions) is not routinely capitalised (see e.g. https://en.wikipedia.org/wiki/Isle_of_Wight).  So your change would turn the current text "This page contains data downloads for the Isle of Wight" (right) to "This page contains data downloads for The Isle of Wight" (not right).  sorry, is a picky point... [we deliberately never start a sentence with the region, so it can be always lower case]

I agree that removing the unwanted space is better, thanks. so perhaps just modify the capital 'The' to small 'the' in lines 8-10? then I am happy :) and I will update pct-scripts accordingly

...thanks for adding the 'the' switch function

Reply:
Right, I understand now.  Thanks.